src/circuit/gadget/sinsemilla/merkle.rs

```diff
@@ -16,12 +16,8 @@ pub(in crate::circuit) mod chip;
 /// SWU hash-to-curve personalization for the Merkle CRH generator
 pub const MERKLE_CRH_PERSONALIZATION: &str = "z.cash:Orchard-MerkleCRH";
 
-/// $\mathsf{MerkleDepth^{Orchard}}$
-pub(crate) const MERKLE_DEPTH_ORCHARD: usize = 32;
-
-/// $\ell^\mathsf{Orchard}_\mathsf{base}$
-/// Number of bits in a Pallas base field element.
-pub(crate) const L_ORCHARD_BASE: usize = 255;
+/// Depth of Merkle tree
+pub(crate) const MERKLE_DEPTH: usize = 32;
```

Comment:
Is this hard-coded in the Merkle gadget? I don't see it used anywhere other than code comments (where yes it makes sense to rename it) and in the tests, which suggests this should be moved into the testing code and not be a constant here.

Reply:
This has been moved into the testing code.